src/schemas/user.schema.ts

```diff
@@ -2,9 +2,10 @@ import { z } from 'zod';
 
 export const userSchema = z.object({
   id: z.string().uuid(),
-  name: z.string(),
-  email: z.string().email(),
-  password: z.string().min(8),
+  name: z.string({ required_error: 'Name is required'}),
+  email: z.string().email({ message: 'Invalid email address'}),
+  password: z.string().min(8, { message: 'Password must be at least 8 characters long'}),
+  deleted: z.boolean.optional(),
```

Comment:
the `z.boolean` is missing the parentheses, but at the same time, i think this could be a mandatory field
```suggestion
  is_deleted: z.boolean(),
```